types/time.go

```diff
@@ -2,12 +2,16 @@ package types
 
 import (
 	"fmt"
-	"math"
 	"strconv"
 	"strings"
 	"time"
 )
 
+const (
+	onePadding = "0"
+	twoPadding = "00"
```

Comment:
🥃 I don't think there's an efficiency or legibility benefit from these constants; They're used once, and I think it's the same to the compiler either way. I wouldn't be adverse to inlining them

Reply:
Yeah good point, I had this because I had a big list at the start then refactored. 🚀